sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -4,8 +4,10 @@
 
 EAPI=6
 
+: ${CMAKE_MAKEFILE_GENERATOR:=ninja}
```

Comment:
@zzlei, could you do this? I'd rather not do another pull request against your pull request for this ;-). Something along the lines:

    # Ninja provides better scalability and cleaner verbose output, and is used throughout all LLVM projects.